docs/guide/tutorials/connect/authentication.txt

```diff
@@ -1,3 +1,236 @@
 ==============
 Authentication
 ==============
+
+The Node.js driver supports all MongoDB :manual:`authentication mechanisms </core/authentication/>` , including those only available in the MongoDB :manual:`Enterprise Edition </administration/install-enterprise/>` .
+
+DEFAULT
+-------
+
+To use the default mechanism, either omit the authentication mechanism specification or specify ``DEFAULT`` as the mechanism in the :manual:`URI ConnectionString </reference/connection-string/>` . The driver will attempt to authenticate using the :manual:`SCRAM-SHA-256 authentication </core/security-scram/>` method if it is available on the MongoDB server. If the server does not support ``SCRAM-SHA-256``, but does support `SCRAM-SHA-1``, the driver will authenticate with ``SCRAM-SHA-1``. If neither ``SCRAM-SHA-256`` or ``SCRAM-SHA-1`` are supported on the server, the driver will authenticate using ``MONGODB-CR`` .
+
+Include the name and password and the :manual:`authentication database </core/security-users/#user-authentication-database>` (\ ``authSource``\ ) in the connection string.
+
+In the following example, the connection string specifies the user ``dave``\ , password ``abc123``\ , and authentication mechanism ``DEFAULT``.
+
+.. important::
+   The user and password should always be **URI** encoded using ``encodeURIComponent`` to ensure any non URI compliant user or password characters are correctly parsed.
+
+.. code-block:: js
+
+   const { MongoClient } = require('mongodb');
+
+   const user = encodeURIComponent('dave');
+   const password = encodeURIComponent('abc123');
+   const authMechanism = 'DEFAULT';
+
+   // Connection URL
+   const url = `mongodb://${user}:${password}@localhost:27017/?authMechanism=${authMechanism}`;
+
+   // Create a new MongoClient
+   const client = new MongoClient(url);
+
+   // Use connect method to connect to the Server
```

Comment:
Not required for lgtm, but can this comment be changed to:
"// Use connect method to connect to the Mongodb server" for clarity